README.md

```diff
@@ -195,6 +195,23 @@ We appreciate all your contributions. If you plan to:
 Please see the [contribution guide](CONTRIBUTING.md) for more information.
 
 
+<!-- CITATION -->
+### Citation
+
```

Comment:
If you use PIQ in your project, please, cite it as follows.

Reply:
Added this